consensus/src/liveness/round_state.rs

```diff
@@ -235,6 +235,8 @@ impl RoundState {
         }
         let new_round = sync_info.highest_round() + 1;
         if new_round > self.current_round {
+            self.pending_votes.log_voting_power();
```

Comment:
Hmm you probably want to check if this is the leader and log the voting power in that case? Otherwise, all other nodes will log 0, which will be hard for us to distinguish the case for when leader didn't receive the vote vs non-leaders logging 0.

Reply:
I don't have that here easily. but I'll just skip in log_voting_power if no pending_votes are there (if there are - I am the proposer).

Reply:
I don't have access here to whether I was a proposer or not. 
But I will change log_voting_power to only log if there are any votes - because then I know I am the proposer.